docs/CoseSignTool.md

```diff
@@ -24,6 +24,52 @@ You may also want to specify:
 ~~~
 Import-Certificate -FilePath 'c:\my\cert.pfx' -CertStoreLocation 'Cert:CurrentUser\MyNewStore'
 ~~~
+* Headers:
+*   There are two ways to supply headers: (1) command-line, and, (2) a JSON file. Both options support providing protected and un-protected headers with int32 and string values. The header label is always a string value.
+>Note: When both file and command-line header options are specified, the command-line input is ignored.
+
+    * Command-line:
+        * /IntProtectedHeaders, /iph - A collection of name-value pairs (separated by comma ',') with the value being an int32. Example: /IntProtectedHeaders created-at=12345678,customer-count=10
+        * /StringProtectedHeaders, /sph - A collection of name-value pairs (separated by comma ',') with the value being a string. Example: /StringProtectedHeaders message-type=cose,customer-name=contoso

```

Comment:
Can you tweak the usage example to show the difference in syntax between an int header and a string header?